code/modules/mob/living/simple_animal/abnormality/teth/crumbling_armor.dm

```diff
@@ -120,6 +138,71 @@
 			to_chat(user, "<span class='userdanger'>A strange power flows through you!</span>")
 	return
 
+/mob/living/simple_animal/hostile/abnormality/crumbling_armor/ZeroQliphoth(mob/living/carbon/human/user)
+	MeltdownEffect()
+	datum_reference.qliphoth_change(3)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/crumbling_armor/proc/MeltdownEffect(mob/living/carbon/human/user)
+	var/list/potentialmarked = list()
+	var/list/marked = list()
+
+	for(var/mob/living/carbon/human/L in GLOB.player_list)
+		if(L.stat >= HARD_CRIT || L.sanity_lost || z != L.z) // Dead or in hard crit, insane, or on a different Z level.
+			continue
+		potentialmarked += L
+		to_chat(L, "<span class='danger'>You feel an overwhelming sense of foreboding...</span>")
+
+	SLEEP_CHECK_DEATH(10 SECONDS)
+	for(var/i=numbermarked, i>=1, i--)
+		var/mob/living/Y = pick(potentialmarked)
+		if(faction_check_mob(Y, FALSE) || Y.z != z || Y.stat == DEAD)
+			continue
+		if(Y in marked)
+			continue
+		marked+=Y
+		to_chat(Y, "<span class='userdanger'>Show me that you can stand your ground!</span>")
+		playsound(get_turf(Y), 'sound/abnormalities/crumbling/warning.ogg', 50, 0, 2)
+		Y.apply_status_effect(STATUS_EFFECT_COWARDICE)
+
+//status
+/datum/status_effect/cowardice
+	id = "cowardice"
+	status_type = STATUS_EFFECT_UNIQUE
+	duration = 50		//Lasts 5 seconds
+	alert_type = /atom/movable/screen/alert/status_effect/cowardice
+
+/atom/movable/screen/alert/status_effect/cowardice
+	name = "Cowardice"
+	desc = "Show me that you can stand your ground!"
+	icon = 'ModularTegustation/Teguicons/status_sprites.dmi'
+	icon_state = "crumbling"
+
+/datum/status_effect/cowardice/on_apply()
+	RegisterSignal(owner, COMSIG_MOVABLE_MOVED, .proc/Punishment)
+	return..()
+
+/datum/status_effect/cowardice/on_remove()
+	UnregisterSignal(owner, COMSIG_MOVABLE_MOVED)
+	return..()
+
+/datum/status_effect/cowardice/proc/Punishment()
+	SIGNAL_HANDLER
+	var/mob/living/carbon/human/H = owner
+	if(!istype(H))
+		return
+	var/obj/item/bodypart/head/head = owner.get_bodypart("head")
+	if(!istype(head))
+		return FALSE
+	playsound(get_turf(H), 'sound/abnormalities/crumbling/attack.ogg', 50, 0, 2)
+	H.apply_damage(50, PALE_DAMAGE, null, H.run_armor_check(null, PALE_DAMAGE), spread_damage = TRUE)
+	if(H.health < 0)
+		head.dismember()
+	new /obj/effect/temp_visual/slice(get_turf(H))
+	qdel(src)
```

Comment:
make it a 1 second to deal 20 or 30

Reply:
I'm going to go with kirie's suggestion here. 50 damage is a lot, but this is also extremely easy to avoid if you see it coming. That's the difference between this and the other abnormalities you mentioned. I suspect that at 30 pale for 1 second of wait is far too easy but its probably better all things considered. It can always be buffed later.